chapters/synchronous.tex

```diff
@@ -295,8 +295,8 @@ \subsection{Argument Restrictions (Component Expression)}\label{argument-restric
 \end{lstlisting}
 \end{example}
 
-Note that the operators defined in this chapter do not automatically vectorize,
-but some operate on arrays in a similar way.
+None that the operators defined in this chapter vectorize, but some can pass through an array of clocked expressions (but not of clocks) in a similar way.
```

Comment:
Should it be "None of the"?

Reply:
Wasn't @sjoelund's point here that while it's not sufficient to be an array of (clocked) expressions, it may be ok to operate on an (clocked) array variable?

Perhaps just drop _in a similar way_?

Reply:
Yes.

Reply:
> Should it be "None of the"?

Yes. Changing.

I was switching between "Note that" and "None of"; and got stuck. Clearly "Note that" was wrong, since this is place where we are stating the restriction.

Reply:
Looks good to me. Resolving.